js/index.js

```diff
@@ -4,4 +4,13 @@ const selectGenre = document.getElementById('selectGenre');
 //on change submit form
 selectGenre.addEventListener('change', () => {
     document.getElementById('filterForm').submit();
-});
\ No newline at end of file
+});
+
+//select search bar
+// const searchByText = document.getElementById('textFilter')
+
+//submit on every change
+// searchByText.addEventListener('change', () => {
+//     document.getElementById('textFilterForm').submit();
+
+// })
```

Comment:
Really you should remove commented out code before going into main - comments in code (especially HTML/CSS/JS) increase file size and therefor increase the amount of time the application takes to load